pkg/ibmvpc/examples/out/explain_out/externalToSubnet_all_vpcs_explain_detail.txt

```diff
@@ -0,0 +1,65 @@
+Explaining connectivity from 161.26.0.0 to subnet3-ky within test-vpc1-ky
+Interpreted source: 161.26.0.0 (external)
+Interpreted destination: vsi3a-ky[10.240.30.5], vsi3b-ky[10.240.30.4], db-endpoint-gateway-ky[10.240.30.6]
```

Comment:
Do we want to change that to `Interpreted source(s)` and `Interpreted destination(s)`?

Reply:
I don't think so. The upper header references the original input which is the subnet. Its likely the user knows that the input refers to a subnet (even if we do not explicitly add the term "subnet")

Reply:
It's not about what the used known or doesn't know. It's a language thing: singular vs. plural.
The title says `Interpreted destination`, but then 3 different destinations are listed.
Just a little bit weird, but can stay as-is if you want to.

Reply:
fixed